app/imports/components/apply/applyLayout.jsx

```diff
@@ -0,0 +1,33 @@
+import React, { PropTypes } from 'react';
+
+// Import apply parts
+import TextTest from '/imports/components/apply/text-test.jsx';
+
+import FormTest from '/imports/components/apply/form-test.jsx';
+
+const getApplySection = ( isLogged, user, apply ) => {
+  // Check if logged in
+  if( !isLogged  ) {
+    return <Accounts.ui.LoginForm />;
```

Comment:
That's how you return a React Component. <Accounts> doesn't need to be imported cuz it'a available anywhere.

A react component should return a string, html code and/or react components.

Something to have always in mind is that it should return only one element with as many children is needed. 

👍 :
```
return (
  <div>
    <p>Holiii</p>
    <SomeReactComponent />
  </div>
)
```

👎 :
```
return (
    <p>Holiii</p>
    <SomeReactComponent />
)
```

This errors out on transpile.